sdks/python/examples/lifespans/worker.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, AsyncGenerator, cast
+from uuid import UUID
+
+from psycopg_pool import ConnectionPool
+from pydantic import BaseModel
+
+from hatchet_sdk import Context, EmptyModel, Hatchet
+
+hatchet = Hatchet(debug=True)
+
+
+# ❓ Use the lifespan in a task
+class TaskOutput(BaseModel):
+    num_rows: int
+    external_ids: list[UUID]
+
+
+lifespan_workflow = hatchet.workflow(name="LifespanWorkflow")
+
+
+@lifespan_workflow.task()
+def sync_lifespan_task(input: EmptyModel, ctx: Context) -> TaskOutput:
+    pool = cast(ConnectionPool, ctx.lifespan["pool"])
+
+    with pool.connection() as conn:
+        query = conn.execute("SELECT * FROM v1_lookup_table_olap LIMIT 5;")
+        rows = query.fetchall()
+
+        for row in rows:
+            print(row)
+
+        print("executed step1 with lifespan", ctx.lifespan)
+
+        return TaskOutput(
+            num_rows=len(rows),
+            external_ids=[cast(UUID, row[0]) for row in rows],
+        )
+
+
+@lifespan_workflow.task()
+async def async_lifespan_task(input: EmptyModel, ctx: Context) -> TaskOutput:
+    pool = cast(ConnectionPool, ctx.lifespan["pool"])
```

Comment:
i'm assuming this not typed?

Reply:
yeah, I don't think it's possible unfortunately unless we want to require you to pass the generic to every workflow or something. FastAPI's aren't typed either 🤷